apps/block/google_news_result.ts

```diff
@@ -34,10 +34,10 @@ export class GoogleNewsResult extends SearchResultToBlock {
       return;
     }
 
-    let href = anchor.getAttribute("href") as string;
+    let href = anchor.getAttribute("href");
```

Comment:
I realized here that an early return would be sufficient.
However, there's no issue with this change, so I'll approve it.